### DISCUSSION TITLE
지이현 - 마케팅 - 페이스북 광고 데이터 분석

### DISCUSSION THREAD

**Discussion by ihyunchi** [dinnercoding/wanted-pre-onboarding-challenge-data#101] [2023년 11월 프리온보딩 데이터 챌린지]
# 도메인을 선택한 이유
- 퍼포먼스 마케터로 근무한 경험이 있습니다. 도메인 지식을 살리고자 이 주제를 선정했습니다. 

# 구체적인 문제 정의
- 한정된 예산으로 효율적인 광고 집행 필요
-  광고 타겟의 연령, 성별 분석을 통해 광고 효율성이 높은 타겟층을 도출합니다
-  시계열 데이터 분석을 통해 요일별 트렌드를 확인하고 적정 예산이 분배될 수 있도록 합니다. 
캐글에 공개된 데이터가 14일 정도라 좋은 분석을 할 수 있을지 걱정입니다. 더 좋은 데이터셋을 찾으면 업데이트 하도록 하겠습니다 :)  

# 데이터 업로드
- [Facebook Ad Campaign Dataset](https://www.kaggle.com/datasets/madislemsalu/facebook-ad-campaign)
- 일자, 캠페인, 연령, 성별, 관심사, 노출, 클릭, 광고비, 전환 수 
- kaggle에서 참고한 데이터 입니다.

**Comment by ghost** [comment]
이현님! 마케터 경험을 살려서 비즈니스와 직접적으로 관련된 문제 정의를 해주셨네요~ 
실제 회사 업무였다면 회사에 강력한 impact를 주게 될 데이터 분석일 것이라는 생각이 드네요. 
기대해보겠습니다 ^^
